pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi, pkgs }:
+
+buildPythonApplication rec {
+  pname = "pydeck";
+  version = "0.7.0";
+
+  src = pkgs.python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1zi0gqzd0byj16ja74m2dm99a1hmrlhk26y0x7am07vb1d8lvvsy";
+  };
+
+  doCheck = false; #can't find pytest
+
+  propagatedBuildInputs = with pkgs.python3Packages; [
+    ipykernel
+    ipywidgets
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/visgl/deck.gl/tree/master/bindings/pydeck";
+    description = "pydeck: Large-scale interactive data visualization in Python";
```

Comment:
Descriptions shouldn't be prefixed with the package name.

Reply:
fixed.